src/runtime/tests/manifest-test.ts

```diff
@@ -30,6 +30,16 @@ import {RamDiskStorageKey, RamDiskStorageDriverProvider} from '../storageNG/driv
 import {digest} from '../../platform/digest-web.js';
 import {DriverFactory} from '../storageNG/drivers/driver-factory.js';
 import {RefinementExpression, BinaryExpressionNode} from '../manifest-ast-nodes.js';
+import {RamDiskMemoryProvider} from '../ram-disk-memory-provider.js';
+import {VolatileMemory} from '../storageNG/drivers/volatile.js';
+
+class RamDiskMemoryImpl implements RamDiskMemoryProvider {
```

Comment:
FYI I've broken this out to a testing class in a later PR and renamed it to `TestRamDiskMemoryProvider`.